src/main/java/nextstep/subway/favorite/ui/FavoriteController.java

```diff
@@ -18,22 +21,24 @@ public FavoriteController(FavoriteService favoriteService) {
     }
 
     @PostMapping("/favorites")
-    public ResponseEntity createFavorite(@RequestBody FavoriteRequest request) {
-        favoriteService.createFavorite(request);
+    public ResponseEntity createFavorite(@AuthenticationPrincipal LoginMember loginMember,
```

Comment:
```suggestion
    public ResponseEntity<Void> createFavorite(@AuthenticationPrincipal LoginMember loginMember,
```
이런 식으로 빈 응답객체를 표현해줄 수 있습니다 :)